extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -25,6 +25,32 @@ define('wikia.articleVideo.featuredVideo.tracking', ['wikia.tracker'], function
 		window.guaSetCustomDimension(36, currentVideo.tags);
 	}
 
+	/**
+	 * Comscore Video Metrix tracking
+	 * C1 (identifier of content) = 1
+	 * C2 (client ID) = 6177433 for Fandom
+	 * C5 (video type identifier) = 04 for featured videos
+	 * We need to track at
+	 */
+	function trackComscoreVideoMetrix() {
+		//Do not track to comscore on dev env
+		if (window.wgDevelEnvironment) {
+			return;
+		}
+
+		var scriptId = 'comscoreVideoMetrixTrack',
+			mountedScript = document.getElementById(scriptId);
+
+		if (mountedScript) {
+			mountedScript.parentElement.removeChild(mountedScript)
```

Comment:
why do you remove and append the same script?

Reply:
These are comscore tracking params, you can think of these as of GA params, but predefined by the tracking software vendor. Documentation: https://wikia-inc.atlassian.net/secure/attachment/116280/comScoreImplementationSpecs-Video.pdf

Reply:
That's because we need to retrigger the request at each subsequent video play.